src/state/victory.rs

```diff
@@ -0,0 +1,13 @@
+use notan::prelude::*;
+
+use crate::state::{Stage, State};
+
+pub fn update(app: &mut App, state: &mut State) {
+    state.hover = None;
+
+    if app.keyboard.was_pressed(KeyCode::Space) {
+        state.stage = Stage::Playing;
+        state.elapsed_milisec = 0;
```

Comment:
I'd rename elapsed_milisec to something more applicable to total player/game time

Reply:
Maybe `run_timer_milisec`?